sites/_variables.scss

```
@@ -14,6 +14,11 @@ $search-gray: #e0e0e0;
 $profile-gray: #eaebed;
 $gray-light: #6f6f6f; // Light grey colour (Bootstrap override).
 
+// Brand variables
```

Those brand variable seems to be specific for the main page title. It might be better to change the comment to said for what component those value are created for. 

```suggestion
// Main page title - Brand variables
```

---

Done